public_html/wp-content/mu-plugins/blocks/source/blocks/sessions/session-list.js

```diff
@@ -150,7 +150,9 @@ class SessionList extends Component {
 			];
 		}
 
-		if ( 'session_time' !== sort ) {
+		if ( 'session_time' === sort ) {
+			args.sort = 'title_asc';
+		} else {
```

Comment:
This seems a little bit confusing since there is a separate clause right below this that does a custom sort if `session_time` is the `sort` value. What if instead of essentially sorting twice, we just added more logic to the custom sort function? Something like:

```js
if ( Array.isArray( filtered ) && 'session_time' === sort ) {
	filtered = filtered.sort( ( a, b ) => {
		if ( Number( a.meta._wcpt_session_time ) === Number( b.meta._wcpt_session_time ) ) {
			return a.title.localeCompare(b.title);
		}

		return Number( a.meta._wcpt_session_time ) - Number( b.meta._wcpt_session_time );
	} );
}
```

_(I hadn't heard of `localeCompare` before, but saw it [here](https://stackoverflow.com/a/16481400))_

Reply:
Yeah, this method makes more sense - I've updated the PR to use this in e42055866d54b119843f00bc8677fc8229ef9bb0